pillar/params.sls

```diff
@@ -23,6 +23,9 @@ dns:
 kube_user:        'kube'
 kube_group:       'kube'
 
+# Log level for kubernetes services, default is 2
+log_level:        '2'
```

Comment:
We can use this variable for flanneld as well, it seems that only etcd is using different way of configuration for log_levels. Thanks for comment